code/game/turfs/unsimulated.dm

```diff
@@ -1,6 +1,7 @@
 /turf/unsimulated
 	name = "command"
 	initial_gas = list(/decl/material/gas/oxygen = MOLES_O2STANDARD, /decl/material/gas/nitrogen = MOLES_N2STANDARD)
+	simulated = FALSE
```

Comment:
Woah woah. Nooope. `/turf/unsimulated` is just for lack of ZAS sim, isn't it? While `/turf/unsimulated/dark_filler` doesn't physically exist and so needs `simulated = FALSE`.